app/src/androidTest/java/com/github/h3lp3rs/h3lp/MedicalCardActivityTest.kt

```diff
@@ -30,18 +28,18 @@ import java.util.*
 
 
 @RunWith(AndroidJUnit4::class)
-class MedicalCardAcivityTest {
+class MedicalCardActivityTest {
 
     private val ctx: Context = ApplicationProvider.getApplicationContext()
 
     @get:Rule
     val testRule = ActivityScenarioRule(
-        MedicalCardAcivity::class.java
+        MedicalCardActivity::class.java
     )
     @Test
     fun oldYearNumberLeadToError() {
         onView(withId(R.id.medicalInfoBirthEditTxt))
```

Comment:
Maybe you can store this in a variable to avoid redundancy, same for the  .perform... calls  that you use repeatedly can be extracted to a method but it's just a detail